index.js

```diff
@@ -238,11 +269,18 @@ const decryptResponse = async response => {
         iv: response.data.iv,
       })
       Logger.log(`[HTTP] Decrypted data in: ${Date.now() - decryptionTime}ms`)
-      return {
+      const decryptedResponse = {
         ...response,
         data: JSON.parse(decryptedData),
       }
-    } else if (
+      cache.set(sanitizedPath, {
+        hash: response.data.metadata.hash,
```

Comment:
Did you test if the built in HTTP funcionality had this whole cache mechanism built in?

Reply:
Yeah, I did test that, but in our current use case, Axios caches the *encrypted* response instead of the *decrypted* one. So that way, the app will still decrypt duplicate encrypted responses as well.

The custom behavior that's implemented currently allows for the app to cache the *decrypted* response so once you call the API and the API sends back a duplicate response, we will skip the decryption part altogether and grab the cached decrypted response instead.